jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -303,6 +303,8 @@ pipeline {
                                         testType: "integ-test",
                                         componentName: "${COMPONENT_NAME}",
                                     )
+                                    echo("Download report manifest from https://ci.opensearch.org/ci/dbc/integ-test/${env.version}/${env.buildId}/${env.platform}/${env.architecture}/${env.distribution}/test-results/${env.BUILD_NUMBER}/integ-test/test-report.yml")
+                                    archiveArtifacts artifacts: 'test-report.yml'
```

Comment:
Is this path right? Wondering if we need $WORKSPACE?Were you able to test this?

Reply:
@gaiksaya Yes. I run on our beta jenkins and it's good.